cosmic-applet-theme/src/theme_subscription.rs

```diff
@@ -0,0 +1,62 @@
+// Copyright 2025 System76 <info@system76.com>
+// SPDX-License-Identifier: GPL-3.0-only
+
+use cosmic::iced::{Subscription, futures::SinkExt, stream};
+use notify::{Config, Event, RecommendedWatcher, RecursiveMode, Watcher};
+use std::path::PathBuf;
+use tokio::sync::mpsc;
+
+#[derive(Debug, Clone)]
+pub enum ThemeUpdate {
+    Changed(bool), // is_dark
+}
+
+pub fn theme_subscription(theme_file: PathBuf) -> Subscription<ThemeUpdate> {
```

Comment:
This file can be removed. In the future you'd want to use the application's core to subscribe to a cosmic_config interface.

```rs

self.core.watch_config::<MyAppConfig>("com.github.MyAppName")